.github/workflows/update-browser-version.yml

```diff
@@ -32,3 +32,8 @@ jobs:
           commit-message: Update the browser version
           committer: Browser Automation Bot <browser-automation-bot@google.com>
           author: Browser Automation Bot <browser-automation-bot@google.com>
+          commit-message: "chore: update the pinned browser version"
```

Comment:
or should we use a non-conventional format?

Reply:
We don't have any convention anyways. Anything is fine. Please note you have two `commit-message` entries, currently.

Reply:
I think I will use `build(chrome): ` to match dependabot too.

Reply:
SG